__tests__/GraphBuilder.test.js

```diff
@@ -7,49 +7,60 @@ const rdf = require('rdf-ext')
 describe('GraphBuilder', () => {
   describe('when the state does not have a resourceURI', () => {
     const state = {
-      'resourceTemplate:bf2:Monograph:Work': {
-        rdfClass: 'http://id.loc.gov/ontologies/bibframe/Work',
-        'http://id.loc.gov/ontologies/bibframe/title': {},
-        'http://id.loc.gov/ontologies/bibframe/temporalCoverage': {},
-        'http://id.loc.gov/ontologies/bibframe/note': {},
-        'http://id.loc.gov/ontologies/bibframe/content': {
-          items: [
-            {
-              id: 'http://id.loc.gov/vocabulary/contentTypes/txt',
-              label: 'text',
-              uri: 'http://id.loc.gov/vocabulary/contentTypes/txt',
-            },
-          ],
-        },
-        'http://id.loc.gov/ontologies/bibframe/illustrativeContent': {
-          items: [
-            {
-              id: '2_RmnVrDkk9',
-              label: 'Genealogical tables',
-              uri: 'http://id.loc.gov/vocabulary/millus/gnt',
-            },
-          ],
+      entities: {
+        resourceTemplates: {
+          'resourceTemplate:bf2:Monograph:Work': {
+            resourceURI: 'http://id.loc.gov/ontologies/bibframe/Work',
+          },
+          'resourceTemplate:bf2:Note': {
+            resourceURI: 'http://id.loc.gov/ontologies/bibframe/Note',
```

Comment:
So just to be clear: This is just a stand-in for the entire resource template?

Reply:
right